src/main/java/scmspain/karyon/restrouter/handlers/DefaultKaryonErrorHandler.java

```diff
@@ -54,6 +57,9 @@
 
     } else {
       statusCode.set(HttpResponseStatus.INTERNAL_SERVER_ERROR);
+
+      L.error("Internal server error", throwable);
```

Comment:
I'm not 100% sure about how logger works, but I think that you have to interpolate the argument, otherwise that argument won't printed:

```java
L.error("Internal server error {}", throwable);
```

But I say, I am not completely sure about that.